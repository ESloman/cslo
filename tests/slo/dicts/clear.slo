var map = {"a": 1, "b": 2, "c": 3};
println(map);
map.clear();
println(map);  # empty
