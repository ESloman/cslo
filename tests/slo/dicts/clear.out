dict[3]: {c: 3, a: 1, b: 2, }
dict[0]: {}
